docs/appendices/release-notes/unreleased.rst

```diff
@@ -241,4 +241,15 @@ Fixes
   and/or broken replica shards when ingesting into a table using a number type
   column with explicit turned off indexing by declaring ``INDEX OFF``.
 
+- Fixed an issue which caused ``INSERT INTO... SELECT...`` statements to fail
+  on insertion of a correct value into partitioned column of a none-string type
+  with a ``CHECK`` constraint.
```

Comment:
Did it really fail? I thought it was excluding these records.

E.g. "Fixed an issue which prevented ``INSERT INTO ... SELECT ..`` from inserting any records if the target table had a partitioned column with a ``CHECK`` constraint on a column of none-string type."